package/autoyast2.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Mar 15 15:54:52 CET 2019 - schubi@suse.de
+
+- Disabling local repositories in second stage only. (bsc#1127818)
```

Comment:
NP: I'd say something like "at the end of 2nd stage when second stage is being run"